app/component/search/search.scss

```diff
@@ -323,7 +299,25 @@ div#search-panel-close-tab {
 .search-header {
   height: 45px;
   line-height: 45px;
-  color: $gray;
+
+  .search-header__category-text {
+    color: $gray-blue;
+    font-size: 11px;
+    margin-left: 12px;
+    margin-right: 6px;
+    vertical-align: middle;
+  }
+
+  .search-header__back-arrow {
+    position: absolute;
+    color: $gray-blue;
+  }
+
+  .search-header__button {
+    margin-bottom: 0;
+    letter-spacing: -0.025em;
+  }
+
 }
 
```

Comment:
Tab bar is missing "blue top border" for selected as in design: https://projects.invisionapp.com/share/5X5RNXGGU#/screens/130683408